### PR TITLE
support/detox change language test

### PR DESCRIPTION
**Pull request by alalmi-ledger**
### 📝 Description

Adding Detox test that changes language on LLM from English to French

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { device, expect } from "detox";
+import PortfolioPage from "../models/wallet/portfolioPage";
+import SettingsPage from "../models/settings/settingsPage";
+import GeneralSettingsPage from "../models/settings/generalSettingsPage";
+import { loadConfig } from "../bridge/server";
+import { delay } from "../helpers";
```
Comment (ggilchrist-ledger) [inline apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts:6]: I think you can remove this

File: apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import { device, expect } from "detox";
```
Comment (ggilchrist-ledger) [inline apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts:1]: Do you need the device import?

File: apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+describe("Change Language", () => {
+  beforeAll(async () => {
+    await loadConfig("1AccountBTC1AccountETH", true);
+    portfolioPage = new PortfolioPage();
+    settingsPage = new SettingsPage();
+    generalSettingsPage = new GeneralSettingsPage();
+  });
+
+  it("should open on Portofolio page", async () => {
```
Comment (ggilchrist-ledger) [inline apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts:20]: I'd remove this step. It's tested elsewhere and not relevant to this language test

File: apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts (hunk 2 above)
Comment (alalmi-ledger) [reply apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts:1]: I think i do? at least i'm getting some error on VSC if i remove it

File: apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts (hunk 1 above)
Comment (alalmi-ledger) [reply apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts:6]: i think i can't

File: apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts (hunk 3 above)
Comment (alalmi-ledger) [reply apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts:20]: ok

Comment (ggilchrist-ledger) [reply apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts]: I can't see it being used elsewhere though

File: apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts (hunk 1 above)
Comment (ggilchrist-ledger) [reply apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts:7]: It's not being used anywhere in this file though. Try removing it and rerunning the test

File: apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts (hunk 2 above)
Comment (ggilchrist-ledger) [reply apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts:1]: It's not being used anywhere in this file though. Try removing it and rerunning the test

**Review by ggilchrist-ledger — APPROVED** [review]

**Pull request merged**